src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { Switch, Route } from "react-router-dom";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+   render() {
+      return (
+         <div className="App">
+            <Switch>
+               <Route path="/" exact />
+               <Route path="/favorites" />
+               <Route path="/search/searchId" />
+               <Route path="/photo/:photoId" />
+               <Route path="/user/:userId" exact />
+               <Route path="/user/:userId/collections" />
```

Comment:
<Route path="/user/collections/:userId" />

Perhaps this will be better, same with the rest